src/placeos-rest-api/session.cr

```diff
@@ -367,26 +361,43 @@ module PlaceOS
 
         ws = driver.debug
         ws.on_message do |message|
-          level, message = Tuple(::Log::Severity, String).from_json(message)
-
-          respond(
-            Response.new(
-              id: request_id,
-              module_id: module_name,
-              type: Response::Type::Debug,
-              level: level,
-              message: message,
-              meta: {
-                sys:   sys_id,
-                mod:   module_name,
-                index: index,
-                name:  name,
-              },
-            ))
+          begin
+            level_value, message = Tuple(Int32, String).from_json(message)
+            level = ::Log::Severity.from_value(level_value)
```

Comment:
Wouldn't it be better to declare a converter for level so you can do `Tuple(::Log::Severity, String).from_json(message)` directly?

Reply:
It's a tuple so we can't annotate anything. 
Sure it would be nice, but this maintains compatibility with [what's happening here](https://github.com/PlaceOS/driver/blob/b613fd1c9e83eacfe9a893041eddf37ed796a415/src/placeos-driver/logger.cr#L41-L49)

Reply:
And we don't want to monkey patch. I think this is a pretty clear way to go about things.